assets/css/_drawer_tab.scss

```diff
@@ -13,7 +13,6 @@
 }
 
 .c-drawer-tab__tab-button {
-  @include button-transparent;
```

Comment:
Brian requested that the drawerTab not turn green when focused. I didn't know the history of adding this `button-transparent`, so lemme know if this is a no-go

Reply:
We could make this more specific though, and remove this but add it specifically for the ladder pages and shuttle page

Reply:
Oh thanks for reminding me, I left out a crucial detail! I did ask Brian if he would be open to reverting the green tab on the shuttle and ladder pages and he said yes, please. So with that in mind, how does it feel to just remove that style?

Reply:
Hmm, then it has no focus states, which isn't _great_. But it's already not great elsewhere, so sure.